scopes/scope/ui/scope-title/scope-title.tsx

```diff
@@ -10,19 +10,27 @@ type ScopeTitleProps = {
   scopeName: string;
   icon?: string;
   backgroundIconColor?: string;
+  iconClassName?: string;
 } & React.HTMLAttributes<HTMLHeadingElement>;
 
 // temporary fix because the API not return the backgroundIconColor from the scope style.
 const DEFAULT_COLOR = '#babec9';
 
-export function ScopeTitle({ scopeName, icon, backgroundIconColor = DEFAULT_COLOR, className }: ScopeTitleProps) {
+export function ScopeTitle({
+  scopeName,
+  icon,
+  backgroundIconColor = DEFAULT_COLOR,
+  className,
+  iconClassName,
+}: ScopeTitleProps) {
   return (
     <H1 className={styles.title} size="sm">
       <ScopeIcon
         size={32}
         displayName={scopeName.split('.')[1] || scopeName}
         scopeImage={icon}
         bgColor={backgroundIconColor}
+        className={iconClassName}
       />
       <span className={classNames(mutedText, styles.orgName, className)}>{scopeName.replace('.', '/')}</span>
     </H1>
```

Comment:
I think we should move the className to the `H1`, where it should be (the type of this component is `HTMLHeadingElement` so I would assume that the className will be on the heading itself, not the `span`.
that way you would be able to access the img with that className.
but I think its ok to pass the `iconClassName` as well.
we need to check that we dont break the usages of the scope title in symphony.
we can either fix it or at least add a TODO for moving the className to the correct location